app/src/main/res/layout/content_settings_privacy.xml

```diff
@@ -53,7 +53,7 @@
         style="@style/SettingsItemClickable"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        android:text="[l10n] Autofill"
+        android:text="@string/autofillSettingsDisplayName"
```

Comment:
Do we have a guideline when adding strings? Should it be camel case or _separated? I've seen both patterns on the project.

Reply:
Yeah both are used in the codebase. Probably more camel case than anything else.